HttpError.js

```diff
@@ -22,12 +22,22 @@ function HttpError (status, message, options) {
     if (options.debug) {
         this.debug = options.debug;
     }
+
+    // Stack Trace
+    if(Error.captureStackTrace) {// AKA if not IE
```

Comment:
nitpick, don't inline comment, move it right above or right below the if statement.